Student/StudentE2ETests/Grades/DSGradesE2ETests.swift

```diff
@@ -0,0 +1,72 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2022-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Core
+import TestsFoundation
+
+class DSGradesE2ETests: E2ETestCase {
+    func testGradesE2E() {
+        // Seed the usual stuff with 2 assignments
+        let users = seeder.createUsers(2)
+        let course = seeder.createCourse()
+        let student = users[1]
```

Comment:
We could save a line here by removing `let users = seeder.createUsers(2)` and writing `let student = seeder.createUser()` and `let teacher = seeder.createUser()`